content/docs/command-reference/import-url.md

```diff
@@ -336,3 +351,48 @@ $ dvc repro
 Running stage 'prepare' with command:
 	python src/prepare.py data/data.xml
 ```
+
+## Example: Transfer to remote storage
+
+When you have a large dataset in an external location, you may want to import it
+to you project without downloading it to the local file system. The
+`--to-remote` option lets you transfer it directly to
+[remote storage](/doc/command-reference/remote). Let's initialize a DVC project,
+and setup a remote:
+
+```dvc
+$ mkdir example # workspace
+$ mkdir /tmp/dvc-storage # remote storage
+$ cd example
+$ git init
+$ dvc init
+$ dvc remote add local_remote /tmp/dvc-storage
```

Comment:
I think I'll get rid of the "local remote" term here to avoid confusions
```suggestion
$ mkdir /tmp/dvc-storage
$ cd example
$ git init
$ dvc init
$ dvc remote add myremote /tmp/dvc-storage
```